### ISSUE TITLE
Items with aria-hidden="true" (e.g. font awesome icons) should be removed

### ISSUE THREAD

**Issue by gijsk** [mozilla/readability#541]
Filed based on https://bugzilla.mozilla.org/show_bug.cgi?id=1545453 .

We should be able to remove all these nodes.

**Comment by gijsk** [comment]
Testcase: https://tagliala.github.io/fa14899.html

**Comment by gijsk** [comment]
See also: https://github.com/FortAwesome/Font-Awesome/issues/14899
